crates/sui/src/wallet_commands.rs

```diff
@@ -710,13 +710,7 @@ pub async fn call_move(
         .execute_transaction(transaction)
         .await?
         .to_effect_response()?;
-    let cert = response.certificate;
-    let effects = response.effects;
-
-    if matches!(effects.status, SuiExecutionStatus::Failure { .. }) {
-        return Err(anyhow!("Error calling module: {:#?}", effects.status));
-    }
```

Comment:
why do we remove this?

Reply:
Ah, should have added a comment!

This conflates an application-level Move `abort` with a Rust-land `Error`. Raising an `Error` here will prevent you from writing a client that wants to keep working after an aborted Move call (as the surfer does).